src/backend/assets/js/session.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function() {
+  var sideslider = $('[data-toggle=collapse-side]');
+  var sel = sideslider.attr('data-target');
+  var top = $('header[role=banner]').outerHeight();
+  var downButton = $('#down-button');
+  var sessionElem = $('.single-session');
+  var sessionId = Number(sessionElem.attr('id'));
+  var maxNoOfSessions = 1000;
```

Comment:
I thought about the current approach and it is true that it is a little quirky. It works correctly but it is not clean.

No. Any number of sessions can be bookmarked. We are not hard coding the values in the local Storage. Here is what we are doing presently:

* We check whether the `localStorage['sessions']` is defined or not. If the user is opening our app for the first time or has cleared his history, then that field will be undefined.

* If the field is not defined, we declare it and assign it a boolean array of 1000 indices. We now initialize the array to 0. The declaration size of the array and initialization of the array to zero is not necessary because of the dynamic nature of the Array.

* When the user clicks on a session, we find out its `sessionID` and mark its value as true on the same array index. That is, if the user clicks on the session which has an id of 5, then
`localStorage['session][5] = 1`

Now, this is where the little problem arises. The IDs are not sequential. They can be any number. Suppose if it is 2000, then although the array we have defined is of size 1000, `localStorage['sessions'][2000] = 1` will be done. To us, It will continue to work perfectly and we won't see any difference. But we are building up a sparse array behind the scenes(indices from 1001 to 1999 will be missing and value will be undefined) and although it won't be wrong, it will be hard to debug and is not a good code practice.

There might be multiple events so we are trying to make it a global to all. The session id of every session is unique regardless of the event. The best we can do here is to declare `localStorage['sessions']` as an `object` instead of an `array` and use the `id` of a `session` as a `field` in that `object`. That way, we would avoid creating a sparse array and write clean code.

Thanks for pointing out this problem.I think we should make a separate issue for it and solve it. This PR is already getting old and big. Please guide. Thanks!!